test/delegation.spec.ts

```diff
@@ -74,6 +84,8 @@ makeSuite('Delegation', (testEnv: TestEnv) => {
     const delegatee = await aaveInstance.getDelegateeByType(users[1].address, '0');
 
     expect(delegatee.toString()).to.be.equal(users[2].address);
+    expect(await aaveInstance.isSnapshotted(users[1].address, '0')).to.be.equal(true);
+    expect(await aaveInstance.isSnapshotted(users[2].address, '0')).to.be.equal(true);
   });
 
   it('User 1 tries to delegate proposition power to user 3', async () => {
```

Comment:
I would remove "tries to", since it actually does it (same in next case)
same for these cases in delegationWithSnapshot